app/matplotlib.ipynb

```diff
@@ -314,7 +359,14 @@
     "X, Y = np.meshgrid(x, y)\n",
     "Z = X**2 + Y**2\n",
     "\n",
-    "# write your code here\n"
+    "fig = plt.figure(figsize=(10, 8))\n",
+    "\n",
+    "ax = fig.add_subplot(111, projection=\"3d\")\n",
+    "\n",
+    "surf = ax.plot_surface(X, Y, Z, cmap=\"viridis\", linewidth=0, antialiased=False)\n",
+    "ax.zaxis.set_major_formatter('{x:.01f}')\n",
+    "\n",
+    "fig.colorbar(surf, shrink=0.5, aspect=10)\n"
```

Comment:
The 3D surface plot is missing axis labels and a title. You should set the title and labels for the x, y, and z axes as per the task requirements.